### PR TITLE
Maven Central Release

### PR DESCRIPTION
**Pull request by rgladwell**
Update the release process to publish to Maven Central via Sonatype's OSS Nexus repository. As part of this work I also upgraded the SBT configuration.

### CODE REVIEW

**Comment by mariusae** [comment]
This is great, thanks.

File: build.sbt [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name := "mustache.scala"
+
+organization := "me.gladwell"
```
Comment (mariusae) [inline build.sbt:3]: this should be org.monkey, no?

Comment (rgladwell) [inline build.sbt:3]: Yes, the OSS Sonatype repository restricts upload access to group IDs for DNS names user own. I own gladwell.me, so if you wanted to use this pull request you would need to create your own user account registered against your monkey.org domain as detailed here:

https://docs.sonatype.org/display/Repository/Sonatype+OSS+Maven+Repository+Usage+Guide

Sorry, I should have included this in the original merge comment.

**Comment by mariusae** [comment]
Sorry for the delay—I’ve started this process with Sonatype.

**Comment by mariusae** [comment]
OSSRH ticket [here](https://issues.sonatype.org/browse/OSSRH-7537)

**Comment by rgladwell** [comment]
:+1: let me know if you have any problems, its a really complicated process and thanks for doing this

**Comment by rgladwell** [comment]
The OSSRH ticket has now been resolved, would you still like to merge this?

**Pull request closed without merging**